core/src/main/scala/org/apache/spark/Accumulable.scala

```diff
@@ -49,6 +48,7 @@ import org.apache.spark.util.Utils
  * @tparam R the full accumulated data (result type)
  * @tparam T partial data that can be added in
  */
+@deprecated("use AccumulatorV2", "2.0.0")
```

Comment:
NP I'm not in a hurry. Just want to do one more cleanup of build warnings before 2.0.